crds/workspace.devfile.io_devworkspaces.v1beta1.yaml

```diff
@@ -5499,9 +5499,17 @@ spec:
                               - name
                               type: object
                             registryUrl:
+                              description: Registry URL to pull devfile from with
+                                the specified Id in the import reference. It is recommended
+                                to always have the regsitryURL specified if Id is
+                                provided to reference a parent devfile It to provides
+                                a well-defined parent and ensures it gets resolved
+                                consistently in different envrionments.
                               type: string
                             uri:
-                              description: Uri of a Devfile yaml file
+                              description: URI Reference of a Devfile yaml file, can
```

Comment:
Consider changing to:
"URI Reference of a parent devfile YAML file. It can be a full URL or a relative URI with the current devfile as the base URI."

Reply:
updated